src/main/scala/petclinic/models/Owner.scala

```diff
@@ -10,14 +10,14 @@ final case class OwnerId(id: UUID) extends AnyVal
 
 object OwnerId {
 
+  def random: ZIO[Random, Nothing, OwnerId] = Random.nextUUID.map(OwnerId(_))
```

Comment:
Default services aren't in the environment anymore.

Reply:
The version of ZIO required by Quill is making this necessary, but only temporarily. 🤞 Once Quill is updated I can change it.